circle.yml

```diff
@@ -0,0 +1,53 @@
+machine:
+  environment:
+    ADB_INSTALL_TIMEOUT: 5
+
+dependencies:
+  pre:
+    - echo y | android update sdk --no-ui --filter platform
+    - echo y | android update sdk --no-ui --all --filter build-tools-22.0.1
+    - echo no | android create avd --force --name my-android-8 --target android-8 --sdcard 128M

```

Comment:
please use higher target. android-8 is not supported in circle-ci. Please confirm from this link. 
https://circleci.com/docs/android